### ISSUE TITLE
Accessibility TalkBack for Android covers other switches, unable to read other tabs or select them.

### ISSUE THREAD

**Issue by markconnellysm** [splashbyte/animated_toggle_switch#60]
Hi there,

I've got a animated toggle as seen below. With TalkBack turned on, the user is unable to toggle the switches with the reader covering across all three toggles.

Double tapping on the other tabs only re-reads the "Overview" tab name. Semantics widget does not break up the tabs for reader either. The on changed method within the package

Flutter version: 3.22.1
Animated Toggle Switch: ^0.8.2

<img width="344" alt="Screenshot 2024-06-04 at 13 31 44" src="https://github.com/splashbyte/animated_toggle_switch/assets/145022375/c1528e06-7913-4a64-bde5-93a283c2639c">

Thanks,

Mark

**Comment by markconnellysm** [comment]
Add the semantics widget with Button: true, to fix this issue.
